research-software-directory/RSD-as-a-service
# 265 edit related topics (software & project)

# Edit related topics (software & project)

Closes #265 
Closes #89 

Changes proposed in this pull request:
* User can define related software and projects on edit software and edit project pages
* On the software page related projects section is added, before related tools section

How to test:
* `docker-compose down --volumes`: to remove old
* `docker-compose build`: to build all services
* `docker-compose up`: to start
* login with test account, eg. professor1
* **create new software**,
* navigate to "related topics" section
* add related software
* add related projects
* view software page and confirm both sections are shown
* **create new project**,
* navigate to "related topics" section
* add related projects
* add related software
* view project page and confirm both sections are shown

## Example edit project page

![image](https://user-images.githubusercontent.com/9204081/168612381-e22ca070-5fa2-477f-9c67-8a4a6e86b14a.png)

## Example project view page (showing unlocked related projects and tools)

![image](https://user-images.githubusercontent.com/9204081/168612163-78f75b8a-f041-4c29-a407-791880bda421.png)

PR Checklist:
